### ISSUE TITLE
Prepare PROPOSAL for conan 2.0

### ISSUE THREAD

**Issue by Jean1995** [tudo-astroparticlephysics/PROPOSAL#358]
The recipe of PROPOSAL (and also cubicinterpolation) for conan-center-index is not ready for conan2.
Therefore, we have fixed the version of conan that needs to be used for PROPOSAL to 1.x.
This fixed issue #354.

As soon as the package for PROPOSAL and cubicinterpolation has been updated, we need to adapt the files that have been changed with #355.

**Comment by Jean1995** [comment]
This is now officially work in progress with the PRs for the [proposal recipe](https://github.com/conan-io/conan-center-index/pull/16851) and the [cubicinterpolation recipe](https://github.com/conan-io/conan-center-index/pull/16299)

**Comment by Jean1995** [comment]
cubicinterpolation is merged and ready for conan2

**Comment by Jean1995** [comment]
Both the cubicinterpolation and the PROPOSAL recipe are now updated for conan2! 🚀 
This also means that PROPOSAL 7.5.1 is now available on conan-center-index, and we can continue to publish new releases to conan.

The future will show whether there are still bugs in the updeated proposal or cubicinterpolation recipe.

I will close this issue when we have adapted the pip package and documentation (i.e. reverted #355)
